src/adapters/OrganizationsJSONAdapter.js

```diff
@@ -0,0 +1,50 @@
+import {OrganizationsAdapter} from '@webex/component-adapter-interfaces';
+import {Observable} from 'rxjs';
+/**
+ * An Organization object that is a set of people in Webex
+ *
+ * @external Organization
+ * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/OrganizationsAdapter.js#L6}
+ */
+
+/**
+ * @typedef OrganizationsJSON
+ * @param {object} datasource An object that contains organizations keyed by ID
+ * @example
+ * {
+ *  cisco: {
+ *    ID: 'cisco',
+ *    name: 'Cisco',
+ *  },
+ * }
+ */
+
+/**
+ * `OrganizationsJSONAdapter` is an implementation of the `OrganizationsAdapter` interface.
+ * The implementation utilizes a JSON object as its source of membership data.
+ *
+ * @see {@link OrganizationsJSON}
+ * @implements {OrganizationsAdapter}
+ */
+export default class OrganizationsJSONAdapter extends OrganizationsAdapter {
+  /**
+   * Returns an observable that emits the organization object for the given ID.
+   *
+   * @param {string} ID ID of the organization to get
+   * @returns {Observable.<Organization>} Observable stream that emits an organization object
+   */
+  getOrg(ID) {
+    return new Observable((observer) => {
```

Comment:
following @lalli-flores's suggestion, let's use `BehaviorSubject`:
```js
const org = ...find...;
const org$ = org ? new BehaviorSubject(org) : throwError('Could not find...');

return org$;
```

Reply:
@cipak, I'm ok with keeping the `new Observable` format for the JSON adapters because they are simple and complete after one emission. For a new developer looking to understand the code, I feel this format is easier to understand. For the SDK adapters however, because the logic is much more complex, we should use `BehaviorSubject` there

Reply:
all right